### PR TITLE
Update GH templates

### PR DESCRIPTION
**Pull request by thomasheremans**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,32 +1,40 @@
...
+
+### Detailed proposed solution
+<!--Describe the solution to be implemented-->
+
+# 🧪 Tests
+
+<!--Delete any irrelevant item and add other mandatory tests that should be passing-->
+
+- [ ] Works in local docker-compose
+- [ ] Has a attested on a testnet at least once
```
Comment (LukeHackett12) [inline .github/ISSUE_TEMPLATE/feature_request.md:24]:
```suggestion
- [ ] Has attested on a testnet at least once
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -1,32 +1,40 @@
...
-labels: ''
-assignees: ''
+# 🎯 Problem to be solved
 
----
+<!--Describe in detail the problem to be solved by this feature/enhancement and why it is important. -->
+
+# 🛠️ Proposed solution
+
+- [ ] Approved design doc: *link*
```
Comment (LukeHackett12) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]:
```suggestion
- [ ] Approved design doc: [link](*url*)
```

**Review by LukeHackett12 — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!-- 🎉🚀 Thanks for submitting a PR!
+📐 Format this PR template as follows:
+   - 1️⃣ PR title (above): concise overview
+   - 2️⃣ PR body: Replace 'body...' with detailed description of the change.
+   - 3️⃣ category: Pick one, delete the rest.
+   - 4️⃣ ticket: Replace #000 with link to a GitHub issue (or 'none' if PR is trivial).
+🧑‍🎓 Please review our contribution guide https://github.com/ObolNetwork/charon/blob/main/docs/contributing.md
```
Comment (dB2510) [inline .github/pull_request_template.md:7]: charon's contribution guide

Comment (thomasheremans) [reply .github/pull_request_template.md:7]: Isn't it the same for these repos?

**Review by eth2devops — APPROVED** [review]
LGTM

**Review by LukeHackett12 — APPROVED** [review]

**Pull request merged**